build.gradle

```diff
@@ -40,7 +40,7 @@ buildscript {
             fuel          : '1.15.1',
 
             // Kotlin
-            kotlin        : '1.3.11',
+            kotlin        : '1.3.21',
```

Comment:
Old Virgil SDK version used. See few lines above.

Reply:
We're using com.virgilsecurity.crypto:common, com.virgilsecurity.crypto:foundation and com.virgilsecurity.crypto:phe which latest version is 5.0.1:
http://central.maven.org/maven2/com/virgilsecurity/crypto/phe/